gammagl/models/unimp.py

```diff
@@ -0,0 +1,24 @@
+import tensorlayerx as tlx
+import tlx.nn as nn
+from gammagl.layers import MultiHead
+
+class Unimp(tlx.nn.Module):
+    def __init__(self,dataset):
+        super(Unimp, self).__init__()
+
+        out_layer1=int(dataset.num_node_features/2)
+        self.layer1=MultiHead(dataset.num_node_features+1, out_layer1, 4,dataset[0].num_nodes)
```

Comment:
I think users can choose their n_heads, so change this to let users choose their own n_heads instead of a fixed one.